.NET/Microsoft.Recognizers.Text.Sequence/English/Parsers/GUIDParser.cs

```diff
@@ -17,19 +17,19 @@ public class GUIDParser : BaseSequenceParser
 
         private static readonly Regex GuidElementRegex = new Regex(BaseGUID.GUIDRegexElement, RegexOptions.Compiled);
 
-        double ScoreGUID(string GUIDText)
+        public double ScoreGUID(string TextGUID)
         {
             double score = baseScore;
 
-            
-            Match elementMatch = GuidElementRegex.Match(GUIDText);
+
+            Match elementMatch = GuidElementRegex.Match(TextGUID);
```

Comment:
I'll approve and create the MS PR with this changes. However, is this _PascalCased_ parameter name valid for Source Analysis? I thought parameters should be _camelCased_.